test/sequential/test-fs-write-buffer-without-length.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+const filename = path.join(common.tmpDir, 'write.txt');
+
+common.refreshTmpDir();
+
+fs.open(filename, 'w', 0o644, common.mustCall(function(err, fd) {
+  if (err) throw err;
+
+  fs.write(fd,
+           Buffer.from('hello'),
+           3,
+           common.mustCall(function(err, written) {
```

Comment:
Agreed. Fixed in https://github.com/nodejs/node/pull/7856/commits/c9d7a7744f56ffc468da53937dc4a84d518d2693

In both tests I had to keep the definition of `cb` inside the other callback because it references `fd` in `fs.closeSync(fd);`.